db/models/user.js

```diff
@@ -58,13 +58,12 @@ class User extends Model {
 
     static updateNewPassword({ id, newPassword }) {
         return this.encryptPassword(newPassword).then(newEncryptedPassword => {
-            return this.update(
-                {
+            return this.findById(id).then(user => {
```

Comment:
This change more directly queries for the specific user and then updates which returns the revised user record (the previous approach returned an obtuse `[1]` value if successful.